pom.xml

```diff
@@ -101,19 +104,19 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
-            <version>3.4.0</version>
+            <version>3.9.3</version>
```

Comment:
negative, 3.9.3 is the LATEST release, there is no any reason to force using the one latest single available version. What if it have bugs (it always has bugs), users will have no chance to downgrade it.

Reply:
I changed back to `3.4.0` and reverted back to an older workflow version.  Did you have any other suggestions?

Reply:
test scopes are not critical

Reply:
Are you recommending I change them?

Reply:
No opinion for test scope, but for hard deps it's bad practice to bump minimal requires dependencies.